### ISSUE TITLE
Observation resource "applies" attribute missing

### ISSUE THREAD

**Issue by travistcummings** [jamesagnew/hapi-fhir#202]
DSTU2 Version 1.1 of Observation resource had a "applies" attribute.  This attribute is not available in version 1.2-SNAPSHOT, but is listed in the FHIR spec. Can you please add it?

**Comment by jamesagnew** [comment]
Hi Travis,

Observation.applies[x] was renamed to Observation.effective[x] in May in
the FHIR sources themselves. See the relevant tracker here:
http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemEdit&tracker_item_id=7636&start=0
and the definition here: http://hl7-fhir.github.io/observation.html

In the 1.2 cycle we will be updating the resource definitions to match the
latest DSTU2 definitions from HL7, which are basically finalized at this
point (last call for changes has already gone out, QA is starting soon).

That said, we haven't applied this change to the SNAPSHOT that's deployed
to Maven yet. Are you building from source or something? (I confirmed by
downloading the latest 1.2 snapshot from here
<https://oss.sonatype.org/content/repositories/snapshots/ca/uhn/hapi/fhir/hapi-fhir-structures-dstu2/1.2-SNAPSHOT/>
and it still had "applies".

FYI to anyone that wants to get a head start on the changes, there is a
branch in Git called "dstu2_final" which has the latest definitions as well
as some other enhancements that have been added to FHIR DSTU2 since the
April ballot. This branch will be merged into master at some point in the
next ~4 weeks, at which point most of the public reference servers will
also be updated.

Cheers,
James
